frontend/fingertips-frontend/components/organisms/PopulationPyramid/createChartOptions.ts

```diff
@@ -152,11 +161,20 @@ const createAdditionalChartSeries = (
   const series: Array<SeriesOptionsType> = [];
 
   if (dataForGroup) {
+    const femaleSeries = computeDataPercentages(
+      dataForGroup.femaleSeries,
+      dataForGroup.total
+    );
+    const maleSeries = computeDataPercentages(
```

Comment:
I suggest we rename these to `femaleGroupSeries` and `maleGroupSeries` so it doesn't get confused with the series above.